.github/workflows/push-to-algolia-doc.yml

```
@@ -0,0 +1,28 @@
+name: Push specs and snippets to Algolia doc
+
+on:
+  workflow_dispatch:
+    inputs:
+      fake_input:
+        description: input needed to satisfy the yaml linter
+        required: false
```

```suggestion
on: workflow_dispatch
```
on: workflow_dispatch